test/DatabaseIntegrationTest/Tools/SQLExecutor.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+#include "Commands/ICommand.h"
+
+
+namespace seed_cpp { namespace model {
+	class User;
```

Comment:
I don't think you need User for anything here

Reply:
That was a leftover. Removed as suggested